legal-api/tests/unit/api/test_business_filings/test_filings.py

```diff
@@ -111,6 +111,24 @@ def test_get_one_business_filing_by_id(session, client, jwt):
     assert rv.json['filing']['business'] == ar['filing']['business']
 
 
+def test_get_one_business_filing_by_id_raw_json(session, client, jwt):
+    """Assert that the business info cannot be received in a valid JSONSchema format."""
```

Comment:
This doesn't seem to reflect the test.

Reply:
Yeah. It was just a try. Since there is no difference in json to identify whether its a raw or json I left it as is. Do you have any suggestion to test this scenario or should I remove it?

Reply:
It'll fail linting without a comment. I'd suggest something like.
Assert that the raw json originally submitted is returned.

Reply:
ok